global-api/routes/main.py

```diff
@@ -0,0 +1,30 @@
+from fastapi import FastAPI
```

Comment:
Perhaps move this file into the global-api folder, so it's easy to find the program entry point.